docs/fsharp/tutorials/asynchronous-and-concurrent-programming/async.md

```diff
@@ -1,213 +1,360 @@
+### Async.Ignore
 
-*   F# supports data-parallelism in a similar way.
+Creates an asynchronous computation that runs the given computation and ignores its result.
 
- Although it operates very differently, `Async.Parallel` corresponds to `Task.WhenAll` for the scenario of wanting the results of a set of async jobs when they all complete.
+Signature:
 
-### Differences
+```fsharp
+computation: Async<'T> -> Async<unit>
+```
 
-*   Nested `let!` is not allowed, unlike nested `await`
+When to use:
 
- Unlike `await`, which can be nested indefinitely, `let!` cannot and must have its result bound before using it inside of another `let!`, `do!`, or `use!`.
+* When you have an asynchronous computation whose result can be discarded.
+* In conjunction with `Async.Start` or other functions that accept an `Async<unit>`.
 
-*   Cancellation support is simpler in F# than in C#/VB.
+What to watch out for:
 
- Supporting cancellation of a task midway through its execution in C#/VB requires checking the `IsCancellationRequested` property or calling `ThrowIfCancellationRequested()` on a `CancellationToken` object that’s passed into the async method.
+* If you must use this because you wish to use `Async.Start` or another function that requires `Async<unit>`, consider if discarding the result is okay to do. Discarding results just to fit a type signature should not generally be done
 
-In contrast, F# async workflows are more naturally cancellable. Cancellation is a simple three-step process.
+### Async.RunSynchronously
 
-1.  Create a new `CancellationTokenSource`.
-2.  Pass it into a starting function.
-3.  Call `Cancel` on the token.
+Runs an asynchronous computation and awaits its result on the calling thread. This call is blocking.
 
-Example:
+Signature:
 
 ```fsharp
-open System.Threading
+computation: Async<'T> * timeout: ?int * cancellationToken: ?CancellationToken -> 'T
+```
+
+When to use:
+
+* If you need it, use it only once in an application - at the entry point for an executable.
+* When you don't care about performance and want to execute a set of other asynchronous operations at once.
+
+What to watch out for:
+
+* This function blocks the calling thread, which undoes the use of asynchrony if it is used to only execute a single `Async<'T>`
+
+## Interoperating with .NET
+
+You may be working with a .NET library or C# codebase that uses [async/await](../../../standard/async.md)-style asynchronous programming. Because C# and the majority of .NET libraries use the <xref:System.Threading.Tasks.Task%601> and <xref:System.Threading.Tasks.Task> types as their core abstractions rather than `Async<'T>`, you must cross a boundary between these two approaches to asynchrony.
+
+### How to work with .NET async and Task<T>
```

Comment:
One question you don't answer here: In F#, if I call a .NET API that returns `Task`, or `Task<T>`, does that return a "hot" task?

Reply:
that's a good question which encapsulates it all - if there's a FAQ/intro section it can be asked as a rhetorical device, that might work (I've not done a re-read since the initial drop - feel free to ping me when it reaches a point where that'd be appreciated).

(Essentially that's dictated by the method you call - they'll typically be hot as things returning `Task`s will be. When one uses an `|> Async.AwaitTask` in the body of an `async {}` CE, you're just adding a step into the builder to call the method in question (to kick off the work) and wrapping the result wrt handling the completion continuations; each time one instantiates/starts the `Async<T>` it'll traverse and call the method to start the task at the relevant part in the flow)

(Just typing al this in case it happens to overlap with what you mean ;) )